modules/artifact/src/test/java/org/mule/runtime/module/artifact/classloader/ResourceReleaserTestCase.java

```diff
@@ -171,6 +178,36 @@ public void createsInstanceOnlyOnce() throws IOException {
     }
   }
 
+  @Test
+  public void stopOracleDriverLeakyThreads() throws Exception {
+    JdbcResourceReleaser resourceReleaser = new JdbcResourceReleaser();
+
+    Method killOracleThreadsMethod = resourceReleaser.getClass().getDeclaredMethod("disposeOracleDriverThreads");
```

Comment:
don't do this. The runtime will use this by calling the release() method. Therefore, simply call the release method